WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/utils/StatsUtils.kt

```diff
@@ -132,6 +134,36 @@ class StatsUtils
             number
         }
     }
+
+    fun getEntryContentDescriptions(
```

Comment:
It seems this method is made for chart entries - we might want to consider renaming it so the intention/usage is more clear. Wdyt?

Reply:
Good observation! I will rename it!

Reply:
resolved in 9b5f428